streams/cr.m3u

```diff
@@ -102,3 +130,7 @@ https://59ef525c24caa.streamlock.net/vmtv/tvvintage/playlist.m3u8
 https://59ef525c24caa.streamlock.net/vmtv/vmlatino/playlist.m3u8
 #EXTINF:-1 tvg-id="VoiceOverRadioTV.cr",VoiceOver Radio TV (720p)
 https://cloudvideo.servers10.com:8081/8198/index.m3u8
+#EXTINF:-1 tvg-id="XpressoJovenRadio.cr",Xpresso Joven Radio (720p)
+https://stmv.streamingvip.click/xpressojovenradiotv/xpressojovenradiotv/playlist.m3u8
```

Comment:
Geo-blocked? Not 24/7 Couldn't verify

Reply:
Working for me

Reply:
Is Geo-blocked then

Reply:
Not sure about that because I'm in Spain.

Reply:
Nvm, it's antiRussian measure. Works in Japan and Germany